cluster/gce/gci/configure-helper.sh

```diff
@@ -2773,6 +2923,11 @@ function setup-node-termination-handler-manifest {
     fi
 }
 
+function setup-apiserver-network-proxy-manifest {
+    local -r manifest="/etc/kubernetes/addons/apiserver-network-proxy/daemonset.yaml"
+    sed -i "s|__APISERVER_IP__|${KUBERNETES_MASTER_NAME}|g" "${manifest}"
```

Comment:
I'm not convinced this value will route to apiserver in all cases, what is it concretely?